tensornetwork/contractors/custom_path_solvers/nconinterface_test.py

```diff
@@ -0,0 +1,146 @@
+# Copyright 2019 The TensorNetwork Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+import pytest
+# pylint: disable=line-too-long
+from tensornetwork.contractors.custom_path_solvers.nconinterface import ncon_solver, ncon_to_adj, ord_to_ncon, ncon_cost_check
```

Comment:
Can we start doing 
```
... import (stuff,
            more_stuff)
```

Instead of disabling line too long?

Reply:
Sure, no problem. I only copied the former from Martins code (so it is his fault...)

Reply:
Is there a preferred way to do this when the format you suggest would still push over the line limit?  (given the `from tensornetwork.contractors.custom_path_solvers.pathsolvers` already takes us close...)

Reply:
If you're just importing a single thing then you can disable the line too long. Do the above if you want to import multiple things.